### DISCUSSION TITLE
Gammapy.time.robust_periodogram

### DISCUSSION THREAD

**Discussion by jonathanestatistica** [gammapy/gammapy#4751] [Help] [answered]
Hello all,

I’m trying to use the function robust_periodogram in my research. However, I just found module gammapy.time in last documentation of gamma.py. Could someone help me to find it? Also, it would help a lot if I could find some example using this function. Thanks

**Comment by registerrier** [comment, marked as answer]
Hello @jonathanestatistica, the gammapy.time subpackage has been removed a long time ago as most of its functionalities were available elsewhere. In particular, astropy.timeseries (https://docs.astropy.org/en/stable/timeseries/index.html) provides support for periodogram algorithms. 
If you are working with lightcurves produced by gammapy, you can export them to astropy binned time series with the `to_table(format="binned-time-series")` command. See here:
https://docs.gammapy.org/1.1/api/gammapy.estimators.FluxPoints.html#gammapy.estimators.FluxPoints.to_table
